cartridges/plugin_gtm/cartridge/templates/default/gtm/gtmScript.isml

```diff
@@ -37,6 +58,14 @@
         return -1;
     }
 
+    function getGA4OrderId(dataLayer) {
+        if ('ecommerce' in dataLayer && 'transaction_id' in dataLayer.ecommerce) {
+            return dataLayer.transaction_id;
```

Comment:
Is this right?  Conditional makes it seem that it should be `dataLayer.ecommerce.transaction_id`

Reply:
Good catch, this has been fixed